docs/performance_tuning.md

```diff
@@ -0,0 +1,285 @@
+    Triton's [Perf Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) tool specifically fits this purpose. 
+    Here is a simplified output for demonstration purposes:
+
+    ```
+    # NOTE: "my_model" represents the model currently being served by 
+    #       Triton in the background from the previous step.
+
+    $ perf_analyzer -m my_model
+
+    ...
+
+    Inferences/Second vs. Client Average Batch Latency
+    Concurrency: 1, throughput: 482.8 infer/sec, latency 12613 usec
+    ```
+
+    - This gives us a sanity test that we are able to successfully form input requests and receive output responses to communicate with the model backend via Triton APIs. 
+    If Perf Analyzer fails to send requests and it is unclear from the error how to proceed, then you may want to sanity check that your model `config.pbtxt` inputs/outputs match what the model expects. 
+    If the config is correct, check that the model runs successfully using its original framework directly. 
+    If you don't have your own script or tool to do so, [Polygraphy](https://github.com/NVIDIA/TensorRT/tree/main/tools/Polygraphy) is a useful tool to run sample inferences on your model via various frameworks. 
+    Currently, Polygraphy supports ONNXRuntime, TensorRT, and TensorFlow 1.x.
+    - The definition of "performing well" is subject to change for each use case. 
+    Some common metrics are throughput, latency, and GPU utilization. 
+    There are many variables that can be tweaked just within your model configuration (`config.pbtxt`) to obtain different results.
+    - As your model, config, or use case evolves, [Perf Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) is a great tool to quickly verify model functionality and performance.
+
+3. How can I improve my model performance?
+    - To further understand the best model configuration you can provide to Triton for your use case, Triton's [Model Analyzer](https://github.com/triton-inference-server/model_analyzer) tool can help. 
+    After running Model Analyzer to find the optimal configurations for your model/use case, you can transfer the generated config files to your [Model Repository](https://github.com/triton-inference-server/server/blob/main/docs/model_repository.md). 
+    Model Analyzer provides a [Quickstart](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/quick_start.md) guide with some examples to walk through.
+    - Upon serving the model with the newly optimized configuration file found by Model Analyzer and running Perf Analyzer again, you should expect to find better performance numbers in most cases compared to a default config.
+    - Some parameters that can be tuned for a model may not be exposed to Model Analyzer since they don't apply to all models. 
```

Comment:
Are these parameters set somewhere other than the config.pbtxt? MA can manually run through anything in config.pbtxt if you tell it to.

Reply:
Added reference and links to manual config search